_rules/sequentially-focusable-element-has-visible-focus-oj04fd.md

```diff
@@ -71,23 +55,23 @@ WCAG does not have any requirement of how big or small focus indicator should be
 
 #### Passed Example 1
 
-The first [focusable][] element is part of [sequential focus navigation][]. The set of pixels that change color when it is [focused][] (due to default User Agent's styling) is unique. The second [focusable][] element is not applicable because it has been removed from [sequential focus navigation][] due to the negative value for the `tabindex` attribute. Its presence is nonetheless enough to make the first one applicable.
+The first [focusable][] element is part of [sequential focus navigation][]. There is at least one pixel that changes color when it is [focused][] (due to default User Agent's styling). The second [focusable][] element is not applicable because it has been removed from [sequential focus navigation][] due to the negative value for the `tabindex` attribute. Its presence is nonetheless enough to make the first one applicable.
```

Comment:
I don't think this "at least one pixel" phrase needs to be in these descriptions. We don't want to repeat the expectation. I think the following would be sufficient:

```suggestion
The first [focusable][] element is part of [sequential focus navigation][]. It gets an outline focused because of the User Agent's default styling. The second [focusable][] element is not applicable because it has been removed from [sequential focus navigation][] due to the negative value for the `tabindex` attribute. Its presence is nonetheless enough to make the first one applicable.
```

The same comment applies to other exampled.